config/default.json

```diff
@@ -4,6 +4,7 @@
   "cacheContractWrappers": true,
   "logLevel": 9,
   "estimateGas": false,
+  "defaultGasLimit": 4543760,
```

Comment:
why this value and not other (e.g 4543759) ?
Is there a good reason for that except that "truffle use that" ?

Reply:
![quote-there-is-nothing-without-reason-gottfried-leibniz-126-13-19-2](https://user-images.githubusercontent.com/5994398/45596295-bd912480-b9c2-11e8-88cd-4693d3386421.jpg)

Reply:
lets find out the reason or set it to max gas limit